11ty/definitions/idiot.md

```diff
@@ -1,8 +1,24 @@
 ---
 title: Idiot
 slug: idiot
-defined: false
+defined: true
+speech: noun
+exerpt: previously a psychiatric term for someone with a profound intellectual disability; typically used as a generic exclamation of frustration at the actions of another person
 flag:
   level: avoid
   text: 'Ableist language'
+reading:
+  - text: 'The Clinical History of "Moron," "Idiot," and Imbecile'
+    href: https://www.merriam-webster.com/words-at-play/moron-idiot-imbecile-offensive-history
 ---
+Since medieval times, the word "idiot" was used to describe an ignorant person, and from the 14th century specifically someone with intellectual and developmental disabilities (I/DD). 
```

Comment:
We typically lead the definition with the clear, concise definition. I'd repeat what you have in the excerpt here first. Then, I'd add  a header, like ## Etymology or ##  Context.